### DISCUSSION TITLE
Welcome to the Foundations of Information discussions!

### DISCUSSION THREAD

**Discussion by amyjko** [amyjko/foundations-of-information#6] [General]
## 👋 Welcome!
I'm using discussions to help facilitate any discourse about the contents of the book, as well as sharing about how others are using it. I hope you:

  * Share ideas for improvements and additions.
  * Share your feedback on the book.
  * Share how you're using the book.
  * Engage with other community members.

Don't be nasty to each other or to me.

**Comment by lauggh** [comment]
Hi Amy! I'm excited that you've done this book on GitHub and made it freely available. I will be adding this to the World Information Architecture Digest next month to share as a resource for people.

**Reply by amyjko** [reply]
Excellent, thank you for sharing it!
